app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -97,6 +97,7 @@ def attribute_asserter(principal)
       name_id_format: name_id_format,
       authn_request: saml_request,
       decrypted_pii: decrypted_pii,
+      session: user_session,
```

Comment:
can we rename the param to `user_session` also, so it's clear what we're working with? Sometimes we pass around `sp_session` so I would like to be as clear as possible if we can